processors/tenantidprocessor/tenantidprocessor.go

```diff
@@ -63,3 +90,58 @@ func (p *processor) addTenantIdToSpans(traces pdata.Traces, tenantIDHeaderValue
 		}
 	}
 }
+
+func (p *processor) addTenantIdToMetrics(metrics pdata.Metrics, tenantIDHeaderValue string) {
+	rms := metrics.ResourceMetrics()
+	for i := 0; i < rms.Len(); i++ {
+		rm := rms.At(i)
```

Comment:
we still need to add the tenant-id to the resource.

Reply:
Added. this will not be used though for now in prometheus-exporter. Also created https://traceableai.atlassian.net/browse/ENG-8519 for updating this later

Reply:
yep, it's just so we don't have to remember to update it in the future.